### ISSUE TITLE
"Cannot read property "unsubscribe" of undefined" on v1.5.0

### ISSUE THREAD

**Issue by Shiroh1ge** [MurhafSousli/ngx-scrollbar#12]
### Bug Report or Feature Request (mark with an `x`)
```
- [ x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] question
```

### OS and Version?
Windows10

### Versions
Angular 5.1

### Repro steps
Route away from a page that uses ngx-scrollbar
 

Hello, we have just updated to 1.5.0 and we are getting some subscription errors on production, whenever we close a route that uses the ngx-scrollbar. Unfortunately I can't give you a proper stack trace, since the code is minified, but the problem is somewhere here: https://vgy.me/SquZVA.png

**Comment by MurhafSousli** [comment]
Fixed in v1.5.5, Thanks for reporting

**Comment by naveendb92** [comment]
Still, I face the same issue.

![image](https://user-images.githubusercontent.com/29221698/36807969-4a1d01ba-1cea-11e8-8134-284ee0a2556a.png)

Kindly help me with this.
